pages/main/faq.md

```diff
@@ -7,7 +7,43 @@ layout: documentation
 
 This page contains some Frequently Asked Questions.
 ## What is AsteroidOS?
-AsteroidOS is an open source Linux distribution that runs on many different smartwatches. It uses [Qt](http://www.qt.io/) and QML to provide the graphical interface.
+AsteroidOS is an open source Linux distribution that runs on many different smartwatches. It uses [Qt](http://www.qt.io/) and QML to provide the graphical interface.\
+One of the main goals of AsteroidOS is to provide users with greater control over their devices and data privacy. The operating system is fully customizable, and users can modify it to meet their specific needs.
+
+## Which features does AsteroidOS provide?
+Currently, AsteroidOS has these features available:
+ - phone notifications
+ - always on display
```

Comment:
I'm not sure about the correctness of `always on display`. Is `Always-on-Display` the correct spelling here?

Reply:
Yes, or "Always-on Display".  In English, when a phrase is used as an adjective, it is typically hyphenated.  So in this case the noun is "Display" and "Always-on" is the modifying adjective.

Reply:
Same for Tilt-to-Wake and Palm-to-Sleep?

Reply:
I think in this case it would likely be more useful to keep them all the same with hyphens between each word.  While not strictly correct grammatically, few people know that rule and having them all the same will likely reduce questions and emphasize that each is a single thing.